src/PyPlot.jl

```diff
@@ -248,14 +248,14 @@ for f in (:surf,:mesh,:plot_surface,:plot_wireframe,:contour3D,:contourf3D)
     @eval begin
         function $f(X::AbstractVector, Y::AbstractVector, Z::AbstractMatrix, args...; kws...)
             m, n = length(X), length(Y)
-            $f(repmat(reshape(X,1,m),n,1), repmat(Y,1,m), Z, args...; kws...)
+            $f(repeat(X',n,1), repeat(Y,1,m), Z, args...; kws...)
```

Comment:
`X'` does a complex conjugation.  You want `transpose(X)`.  Also, to retain 0.6 compatibility, we need:
```jl
repeat(transpose(X), outer=(n,1)), repeat(Y, outer=(1,m))
```